Headlines/src/Controllers/TrendingCardsViewController.swift

```diff
@@ -71,7 +83,14 @@ class TrendingCardsViewController: UICollectionViewController {
         }
         
         flowLayout.minimumLineSpacing = 10
-        flowLayout.itemSize = CGSize(width: collectionViewSize.width - 20, height: 235)
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            let columns: CGFloat = UIApplication.shared.statusBarOrientation.isLandscape ? 3.0 : 2.0
+            flowLayout.itemSize = CGSize(width: ((collectionViewSize.width - 20 - (columns-1)*10) / columns), height: 235)
+            flowLayout.sectionInset = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+        }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f609a77816dcd1d12fd0f37d58964ee47f86c075/Headlines/src/Controllers/TrendingCardsViewController.swift#L90"></span>Else and catch should be on the same line, one space after the previous declaration.</td>
    </tr>
  </tbody>
</table>